parlai/scripts/self_chat.py

```diff
@@ -86,15 +98,32 @@ def _run_self_chat_episode(opt, world, world_logger):
 
 def self_chat(opt):
     random.seed(opt['seed'])
+    partner = opt['partner_model_file']
+    partner_opt_file = opt['partner_opt_file']
 
     # Create agents
     agent1 = create_agent(opt, requireModelExists=True)
-    agent2 = agent1.clone()
+    if not partner:
+        # Self chat with same model
+        agent2 = agent1.clone()
+    else:
+        # Self chat with different models
+        if partner_opt_file:
+            print(f"WARNING: Loading override opts from: {partner_opt_file}")
+            with open(partner_opt_file) as f:
+                partner_opt = json.load(f)
+
+        print(
+            f"WARNING: Setting partner interactive mode to: {opt['interactive_mode']}"
+        )
+        partner_opt['interactive_mode'] = opt['interactive_mode']
```

Comment:
can we do an `opt.get('interactive_mode', True)` here

Reply:
done :)